ChopChop/Models/Ingredient.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import GRDB
+
+struct Ingredient: Equatable {
+    var id: Int64?
+    var ingredientCategoryId: Int64?
```

Comment:
Is it possible to only have this property in `IngredientRecord`?

Reply:
Sure. The properties for the in-model models can all be changed, as long as the mappings are updated from the records.